src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -4818,13 +5052,48 @@ public static String join(final short[] array, final char delimiter) {
      * @since 3.2
      */
     public static String join(final short[] array, final char delimiter, final int startIndex, final int endIndex) {
+        return join(array, String.valueOf(delimiter), startIndex, endIndex);
+    }
+
+    /**
+     * <p>
```

Comment:
The 1st para of a Javadoc is automatically in a paragraph IIRC.

Reply:
I've followed convention from other Javadoc in the file. Fixed.